### ISSUE TITLE
Connection refused

### ISSUE THREAD

**Issue by Vince0511** [pymedusa/Medusa#9109]
**Describe the bug**
After editing the password in config.ini (after being unable to login) and installing de newest update medusa keeps refusing connection.
**To Reproduce**
Connect to medusa

**Expected behavior**
Display the login-screen.

**Screenshots**
Jus the default 'Connection Refused'-message

**Medusa (please complete the following information):**
 - OS: Ubuntu 20.04.1 LTS (GNU/Linux 5.4.0-62-generic x86_64)
 - Branch: Master
 - Commit: [698ca1d]
 - Python version: 3.8.5
 - Database version: unknown

I've tried different browsers, different devices, several reboots, no luck.

**Comment by p0psicles** [comment]
Try starting manually from command line. Like: python3 start.py

**Comment by Vince0511** [comment]
Thanks, butthat didn't work.

I found something when I was starting it manually.

**
Jan 20 08:00:23 ubuntu20 python3[3227]:     raise TypeError(f'{funcname}() argument must be str, bytes, or '
Jan 20 08:00:23 ubuntu20 python3[3227]: TypeError: join() argument must be str, bytes, or os.PathLike object, not 'NoneType'
**
When I looked further I fount issue 8986, and removed my https-cert and https-key from my config. It fixed the issue!
Thank you anyway, You pointed my in the right direction.
